packages/atomic/src/components/common/atomic-icon-button/atomic-icon-button.tsx

```diff
@@ -21,24 +21,28 @@ export class AtomicIconButton {
   @Prop({mutable: true}) public labelI18nKey!: string;
   @Prop({mutable: true}) public icon!: string;
   @Prop({mutable: true}) public buttonRef?: (el?: HTMLButtonElement) => void;
+  @Prop({mutable: true}) public badge?: string;
 
   public render() {
     return (
-      <Button
-        style="outline-neutral"
-        ariaLabel={this.bindings.i18n.t(this.labelI18nKey)}
-        class="p-3"
-        part="button"
-        onClick={this.clickCallback}
-        title={this.tooltip}
-        ref={this.buttonRef}
-      >
-        <atomic-icon
-          icon={this.icon}
-          class="w-4 h-4 shrink-0"
-          aria-hidden="true"
-        ></atomic-icon>
-      </Button>
+      <Host>
+        <Button
+          style="outline-neutral"
+          ariaLabel={this.bindings.i18n.t(this.labelI18nKey)}
+          class="p-3 relative"
+          part="button"
+          onClick={this.clickCallback}
+          title={this.tooltip}
+          ref={this.buttonRef}
+        >
+          <atomic-icon
+            icon={this.icon}
+            class="w-4 h-4 shrink-0"
+            aria-hidden="true"
+          ></atomic-icon>
+        </Button>
+        {this.badge && <span part="badge">{this.badge}</span>}
```

Comment:
Although we don't necessarily need to be accesible for this project, I would add an aria label here just in case we do in the future.

Reply:
Also, I would document this part